Description: A patch for README before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 This is needed by the sample.c file generated from README
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- a/README
+++ b/README
@@ -128,7 +128,7 @@
 
       bitmap = at_bitmap_read(rfunc, fname, NULL, NULL, NULL);
       splines = at_splines_new(bitmap, opts, NULL, NULL);
-      at_splines_write(wfunc, stdout, "", NULL splines, NULL, NULL);
+      at_splines_write(wfunc, stdout, "", NULL, splines, NULL, NULL);
       return 0;
     }
 
